storage.py

```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
-    
+    def set(self, key, value):
+        if key in self.data:
+            self.data[key] = value
```

Comment:
If there is no key in the data, the function set should return `None`, I suppose.

Reply:
I agree with you. I'll fix it.